website/content/en/status/report-2023-01-2023-03/gcc.adoc

```diff
@@ -8,7 +8,7 @@ link:https://gcc.gnu.org/gcc-12/[GCC 12 release series] URL: link:https://gcc.gn
 Contact: Lorenzo Salvadore <salvadore@FreeBSD.org> +
 Contact: Gerald Pfeifer <gerald@pfeifer.com>
 
-The main news this quarter is the cleaning of old GCC versions from the ports tree: this will allow for a more efficient approach of bugs.
+The main news this quarter is the cleaning of old GCC versions from the ports tree: this will allow for a more efficient approach to bugs.
```

Comment:
Also, you might remove the word _for_ …

Reply:
I like the word _for_, I prefer to keep it. But thanks for the correction.